code/modules/research/experimentor.dm

```diff
@@ -425,18 +425,18 @@
 		if(prob(EFFECT_PROB_LOW) && criticalReaction)
 			visible_message("<span class='warning'>[src]'s crushing mechanism slowly and smoothly descends, flattening the [exp_on]!</span>")
 			new /obj/item/stack/sheet/plasteel(get_turf(pick(oview(1,src))))
-		if(linked_console.linked_lathe)
+		else if(linked_console.linked_lathe)

```

Comment:
I messed up with the last experiment, it works differently than others. Fixing it soon.